### ISSUE TITLE
SC2063: False positive with find | grep -E '.*'

### ISSUE THREAD

**Issue by BrianMusson** [koalaman/shellcheck#1214]
#### For bugs
- Rule Id (if any, e.g. SC1000):  SC2063
- My shellcheck version (`shellcheck --version` or "online"):
[ttys010:brian.musson]% shellcheck --version
ShellCheck - shell script analysis tool
version: 0.4.7
license: GNU General Public License, version 3
website: http://www.shellcheck.net

- [X] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

I think it is looking for something in the way of alphanumeric values?

perhaps .* is too open.

I couple this with find, since it returns 1 if nothing is found (instead of find's 0 return code). 

```sh

#!/bin/bash
find /oem -regextype posix-egrep -regex '.*(emwd.pl|emctl)$' -exec ls -lah --full-time "{}" \; | grep -E ".*"
find /oem -regextype posix-egrep -regex '.*(emwd.pl|emctl)$' -exec ls -lah --full-time "{}" \; | grep -E '.*'


```

#### Here's what shellcheck currently says:

SC2063: Grep uses regex, but this looks like a glob.

#### Here's what I wanted or expected to see:

Nothing

**Comment by Dmole** [comment]
Why not just 'grep .'  ?

**Comment by koalaman** [comment]
This might be better written as `grep ""`, but I agree that `".*"` is not an indication that the user is confusing globs and regex. This has now been fixed, thanks!
